main.go

```diff
@@ -108,6 +152,16 @@ func block(host string) {
 			return errors.New("not live yet")
 		}
 
+		// If Enabled perform the ENVOY ISTIO SDS WARMING Check
+		sdsprotection, ok := os.LookupEnv("ENVOY_ISTIO_SDS_WARMING_PROTECTION")
+		if ok && sdsprotection == "true" {
+			err := checkEnvoyIstioSDS(host)
+			if err != nil {
+				return err
+			}
+			return errors.New("Envoy Istio SDS check failed")
```

Comment:
Looks like you error in all cases here. Is that what you want?

Reply:
thanks, this should be fixed now